app/controllers/jobs_controller.rb

```diff
@@ -1,2 +1,41 @@
 class JobsController < ApplicationController
+  before_action :set_job, only: :show
+  before_action :job_params, only: :create
+  skip_before_action :authenticate_user!, only: %i[index show]
+
+  def index
+    #geocoding stuff goes here
```

Comment:
You can add a normal index here to test if we can accees the seeded jobs. E.g `@jobs = Job.all`